x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/dashboard_actions.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButtonIcon, EuiPopover, EuiContextMenuItem, EuiContextMenuPanel } from '@elastic/eui';
+import { DASHBOARD_APP_LOCATOR } from '@kbn/deeplinks-analytics';
+import { DashboardLocatorParams } from '@kbn/dashboard-plugin/public';
+import { useKibana } from '../../../utils/kibana_react';
+
+interface DashboardActionsProps {
+  dashboardId: string;
+}
+
+export function DashboardActions({ dashboardId }: DashboardActionsProps) {
+  const {
+    services: {
+      share: { url: urlService },
+    },
+  } = useKibana();
+
+  const [isActionsPopoverOpen, setIsActionsPopoverOpen] = useState(false);
+
+  const handleGoToDashboard = async () => {
+    const dashboardLocator = urlService.locators.get<DashboardLocatorParams>(DASHBOARD_APP_LOCATOR);
+    if (!dashboardLocator) {
+      return undefined;
+    }
+    dashboardLocator.navigate({ dashboardId });
```

Comment:
How should we handle the case, where a dashboard is already deleted?

Reply:
I display only valid dashboards in this [commit](https://github.com/elastic/kibana/pull/219019/commits/cf6cbd7adbf44f1aa71dc76ab6aa150640b6104a). If a linked dashboard is deleted, it won't show up in the list of related dashboards.